packages/patternfly-4/react-tokens/src/templates/d.ts.js

```diff
@@ -9,5 +9,9 @@ module.exports = {
         .map(tokenKey => `${JSON.stringify(tokenKey)}: ${JSON.stringify(token[tokenKey])};`)
         .join(' ');
       return `${acc}export const ${key}: { ${tokenTypeValue} }\n`;
-    }, '')
+    }, ''),
+  getSingleOutputPath: ({ outDir, tokenName }) => join(outDir, `js/${tokenName}.d.ts`),
+  getSingleContent: ({ tokenName, tokenValue }) =>
+    `const ${tokenName}: {${Object.entries(tokenValue).map(([key, value]) =>
```

Comment:
can you add `declare` before the const? This is so TS won't complain and throw this message:
`TS1046: 'declare' modifier required for top level element`